app/src/main/java/org/schulcloud/mobile/data/model/RealmString.java

```diff
@@ -0,0 +1,27 @@
+package org.schulcloud.mobile.data.model;
+
+import io.realm.RealmModel;
+import io.realm.annotations.RealmClass;
+
+@RealmClass
+public class RealmString implements RealmModel
```

Comment:
The definition of RealmList is `public class RealmList<E extends RealmModel> extends AbstractList<E> implements OrderedRealmCollection<E>`, so the default String doesn't work